src/ComposerRequireChecker/FileLocator/LocateFilesByGlobPattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ComposerRequireChecker\FileLocator;
+
+
+use Webmozart\Glob\Glob;
+
+class LocateFilesByGlobPattern
+{
+
+    public function __invoke(array $globPatterns = [], $rootDir) : \Generator
+    {
+        $files = [];
+
+        foreach ($globPatterns as $globPattern) {
+            $files = Glob::glob(rtrim($rootDir, '/') . '/' . $globPattern);
+        }
+
+
+        yield from $files;
```

Comment:
Why this instead of yielding from each loop iteration?

```php
yield from Glob::glob(rtrim($rootDir, '/') . '/' . $globPattern);
```

That would also flatten the iterator